### PR TITLE
Move personas content to "About the Docs" and add signposting from Corda 5.0 landing page

### PR DESCRIPTION
**Pull request by nadinequinr3**
Preview: https://nadine.preview.docs.r3.com/

### CODE REVIEW

File: content/en/about-the-docs/_index.md [hunk 1]
```diff
@@ -59,4 +99,20 @@ From Corda 5.0 onwards, there is one documentation set, but the features that re
...
 
 * **A**: The heading without the 'Enterprise only' icon is the most common and it is used where the content applies to all Corda users.
-* **B**: The ‘Enterprise only' icon is used mostly in headings, indicating that the content under that heading relates to a Corda Enterprise feature. On occasions, the icon may be used inline to identify sentences, paragraphs, or tables that apply to a Corda Enterprise feature.
\ No newline at end of file
+* **B**: The ‘Enterprise only' icon is used mostly in headings, indicating that the content under that heading relates to a Corda Enterprise feature. On occasions, the icon may be used inline to identify sentences, paragraphs, or tables that apply to a Corda Enterprise feature.
+
+## Reporting an Error in the Docs
+The R3 Technical Writing team continuously strive to improve our documentation and overall content strategy. As we make improvements, we would love to get your feedback on what we are doing well, and what we could be doing better. 
+
+Click [here](mailto:docs@r3.com) to email the R3 Technical Writing team. To help us deal with your feedback effectively, please include all of the following:
```
Comment (AnnaK-R3) [inline content/en/about-the-docs/_index.md:107]:
```suggestion
Use the [docs@r3.com](mailto:docs@r3.com) email address to contact the R3 Technical Writing team. To help us deal with your feedback effectively, please include all of the following:
```
Vague link phrasing affects usability and is not very informative. Avoid using "Click here" phrases in the docs.

File: content/en/about-the-docs/_index.md [hunk 2]
```diff
@@ -59,4 +99,20 @@ From Corda 5.0 onwards, there is one documentation set, but the features that re
...
+* Your name
+* Your company name
+* Your email address (if different from the one being used to report)
+* The URL of the documentation page (if it is an issue being reported)
+
+Please include as much detail as you can in your feedback, as it is valuable to us and it will be read and shared with the rest of the team. A ticket is then created for any issues received, which will be quickly triaged and, if necessary, assigned to a Technical Writer to effect changes.
+
+{{< note >}}
+We are working on refining our documentation feedback process. In the near future, you will be able to submit feedback more interactively and directly from the documentation.
+{{</ note >}}
```
[the remark this answers is not in the document]
Comment (nadinequinr3) [reply content/en/about-the-docs/_index.md:118]: @colin-hillis this was your content so I'll let you comment.

File: content/en/about-the-docs/_index.md (hunk 1 above)
Comment (colin-hillis) [reply content/en/about-the-docs/_index.md]: I agree

File: content/en/about-the-docs/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (colin-hillis) [reply content/en/about-the-docs/_index.md:118]: Happy to remove

**Review by yarivus — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**